doc/classes/NavigationMesh.xml

```diff
@@ -95,11 +95,12 @@
 		<member name="agent_radius" type="float" setter="set_agent_radius" getter="get_agent_radius" default="0.5">
 			The distance to erode/shrink the walkable area of the heightfield away from obstructions.
 			[b]Note:[/b] While baking, this value will be rounded up to the nearest multiple of [member cell_size].
+			[b]Note:[/b] The radius must be positive. If the radius is [code]0.0[/code], it won't be possible to fix invalid outline overlaps and other precision errors during the baking process. As a result, some obstacles may be excluded incorrectly from the final navigation mesh, or may delete the navigation mesh's polygons.
 		</member>
 		<member name="border_size" type="float" setter="set_border_size" getter="get_border_size" default="0.0">
 			The size of the non-navigable border around the bake bounding area.
 			In conjunction with the [member filter_baking_aabb] and a [member edge_max_error] value at [code]1.0[/code] or below the border size can be used to bake tile aligned navigation meshes without the tile edges being shrunk by [member agent_radius].
-			[b]Note:[/b] While baking and not zero, this value will be rounded up to the nearest multiple of [member cell_size].
+			[b]Note:[/b] While baking and not [code]0.0[/code], this value will be rounded up to the nearest multiple of [member cell_size].
```

Comment:
Nitpick
```suggestion
			[b]Note:[/b] If this value is not [code]0.0[/code], during baking it will be rounded up to the nearest multiple of [member cell_size].
```

Reply:
Done